frontend/src/common/MultiFileInput.tsx

```diff
@@ -75,22 +75,24 @@ export default function MultiFileInput({
   return (
     <Box sx={{ ...(fullWidth && { width: '100%' }) }}>
       {!readOnly && (
-        <Box mb={2}>
+        <>
           <label htmlFor={htmlId}>
             <Button fullWidth component='span' variant='outlined' disabled={disabled}>
               {label}
             </Button>
           </label>
           <Input multiple id={htmlId} type='file' onInput={handleFileChange} accept={accepts} disabled={disabled} />
-        </Box>
+        </>
+      )}
+      {files.length > 0 && (
+        <Stack spacing={1} mt={2}>
```

Comment:
TIL you can use system props outside of `sx`!

Reply:
Most, if not all, MUI components support system props now :)